erc20-price/proto/v1/erc20_price.proto

```diff
@@ -2,12 +2,17 @@ syntax = "proto3";
 
 package messari.erc20_price.v1;
 
-message ERC20Price {
-  uint64 block_number = 1;
+message Erc20Price {
+  string token_address = 1;
   string price_usd = 2;
-  string token_address = 3;
+  uint64 block_number = 3; // last updated block number
+  enum Source {
+    oracles = 0;
```

Comment:
these should be capitalized: https://developers.google.com/protocol-buffers/docs/style#enums